packages/Angular/Explorer/core-entity-forms/src/lib/generated/Entities/User/user.form.component.html

```diff
@@ -513,7 +513,7 @@
 
                     <mj-tab Name="Resource Permissions" 
                         [Visible]="record.IsSaved" 
-                        [Props]="{EntityRelationshipID: 'DAF47024-4687-EF11-8473-6045BDF077EE'}">
+                        [Props]="{EntityRelationshipID: '4842433E-F36B-1410-8560-0041FA62858A'}">
```

Comment:
@JS-BC  example - one of many - of the changed EntityRelationshipID values - when you redo with a clean 2.11.0 DB this should no longer be the case